components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/internal/AppsCommonDataHolder.java

```diff
@@ -40,6 +41,8 @@ public class AppsCommonDataHolder {
 
     private Set<String> systemApplications = new HashSet<>();
 
+    private boolean isOrganizationManagementEnable;
```

Comment:
shouldn't it be `isOrganizationManagementEnabled` ?

Reply:
Yes, Fixed by - https://github.com/wso2/identity-apps/pull/3524